photon-core/src/main/java/org/photonvision/common/dataflow/networktables/NetworkTablesManager.java

```diff
@@ -97,8 +98,11 @@ private void broadcastConnectedStatusImpl() {
     private void broadcastVersion() {
         kRootTable.getEntry("version").setString(PhotonVersion.versionString);
         kRootTable.getEntry("buildDate").setString(PhotonVersion.buildDate);
-    }
 
+        FMSTable.getEntry("MatchNumber").setDefaultNumber(0);
+        FMSTable.getEntry("MatchType").setDefaultNumber(0);
```

Comment:
Doesn't getNumber(0) do the same thing?

Reply:
Yes, but it seems like a better use case for times when the table didn't exist beforehand.

Reply:
After reviewing the NetworkTables javadoc, it looks like this and `entry.getDouble(0)` do the same thing. IMO we don't need this code? It doesn't hurt but is redundant I think.